src/tools/ScenarioMeasurement/Util/Util.cs

```diff
@@ -6,19 +6,24 @@ namespace ScenarioMeasurement
 
     public class Util
     {
-        [DllImport("User32.Dll")]
-        private static extern long SetCursorPos(int x, int y);
-        
-        
+        // [DllImport("User32.Dll")]
+        // private static extern long SetCursorPos(int x, int y);
+
+
         public static void Init()
         {
-            SetCursorPos(0, 0);
+            // SetCursorPos(0, 0);
```

Comment:
This still needs to work on Windows. Please don't comment this out. We may need to split this into `Util.Windows.cs` and `Util.cs` and make them partial classes?

Reply:
Sure! Why do we need to reset the cursor position to (0,0) each time? Does the same functionality apply to Linux as well? hmm not sure how partial classes can help here, if all files haves to be present at compile time even for Linux.

Reply:
This is only necessary for GUI apps which at this point we only support on Windows. We do this to ensure the cursor won't mistakenly be on a window or control for some iterations. By default Windows will shift the position of new windows down and to the right every time the app is launched. If you leave the cursor in its default position in the middle of the screen eventually you'll end up with an iteration of the measurement that has the mouse over it. This can cause any number of things, including mouse events, which can run different code. This can lead to noise. By moving it to 0,0, we avoid the problem entirely.

We can change the project file to only include the `Windows` or `Linux` file conditionally. That way we only try to find `SetCursorPos` on Windows.